packages/wasi/src/index.ts

```diff
@@ -1224,10 +1224,15 @@ export default class WASIDefault {
               throw e;
             }
           }
-          let realfd
+          let realfd;
+          let isDirectory;
+          try {
+            fs.statSync(full).isDirectory()
```

Comment:
Should this be `isDirectory = ...` ?

Reply:
yes, oops, I thought I had force pushed with a fix for this blunder :P

Reply:
Not sure how I missed that on my review. Thanks for the catch @corwin-of-amber